packages/storybook-vue/stories/components/data-grid/DataGrid.stories.mdx

```diff
@@ -689,7 +689,7 @@ Expected format: comma delimited `string`, eg `'one, two, three'`
             },
         ];
         dataGrid.rows = [
-            ['Apple, Banana', 'Tomato, Cucumber'],
+            [[{content: 'Apple', color: 'red'}, {content: 'Banana', color: 'yellow'}], 'Tomato, Cucumber'],
```

Comment:
I think this looks awkward… no? 🤔 

<img width="825" alt="image" src="https://github.com/telekom/scale/assets/851105/5b41a595-9bff-491c-a2e2-fc63a94aac6d">

I would rather document it above:

> Expected format: comma delimited `string`, eg `'one, two, three'` or `array` of objects with `content` and `color` keys, e.g. `{content: 'Apple', color: 'red'}`

What do you think?

Reply:
@arturo you mean it looks visually awkward and to just document it with some text insted? I did it this way because I thought it would be clear visually and in relation to the code snippet that is shown...